pkg/rhttp/rhttp.go

```diff
@@ -103,12 +103,12 @@ func (s *Server) Start(ln net.Listener) error {
 		return err
 	}
 
-	handler, err := s.getHandler()
+	mux, err := s.getServeMux()
```

Comment:
Can you keep the previous function name? Is more generic than ServeMux.
ServeMux is just an implementation of a handler.

Reply:
Sure, it's done.